Source/MediaInfo/Multiple/File_Mpeg4_Elements.cpp

```diff
@@ -1677,10 +1677,11 @@ void File_Mpeg4::ftyp()
                                            #if MEDIAINFO_CONFORMANCE
                                                IsCmaf=true;
                                            #endif
-                                           //fall through
+                                           [[fallthrough]]; //fall through

```

Comment:
The comment can be removed, modern syntax analyzers understand the attribute (x3)